client/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.controller.js

```diff
@@ -0,0 +1,50 @@
+angular
+  .module('managerApp')
+  .controller('TelecomTelephonyGuidesCtrl', class TelecomTelephonyGuidesCtrl {
+    constructor($translate, TucToastError, TELEPHONY_GUIDES) {
+      this.$translate = $translate;
+      this.constant = { TELEPHONY_GUIDES };
+    }
+
+    $onInit() {
+      this.loading = {
+        init: false,
+      };
+      this.guides = null;
+      this.language = null;
+      this.count = null;
+
+      this.guides = this.constant.TELEPHONY_GUIDES;
+      if (localStorage['univers-selected-language']) {
+        this.language = localStorage['univers-selected-language'].replace(/-.*$|_.*$/, '');
+      } else if (navigator.language || navigator.userLanguage) {
+        this.language = (navigator.language || navigator.userLanguage).replace(/-.*$|_.*$/, '');
+      }
+      this.countGuides();
+    }
+
+    /**
+     * Count guides.
+     */
+    countGuides() {
+      if (_.has(this.guides, 'sections')) {
+        _.forEach(this.guides.sections, (section) => {
+          if (_.has(section, 'guides')) {
+            _.forEach(section.guides, (guide) => {
+              if (_.has(guide, ['url', this.language]) && _.isString(guide.label)) {
+                this.count += 1;
+              }
+            });
+          }
+        });
```

Comment:
```suggestion
         this.count = _.flatmap(this.guides.section, section => section.guides).filter(guide => _.has(guide, `url.${this.language}`) && _.isString(guide.label)).length
```

Reply:
@Steffy29 What I meant is that you can replace the code above with the `if` and `forEach` by this

Reply:
No, _.flatMap is not available for our version of lodash

Reply:
You can therefore `map` and then `flat` the resulting array